src/containers/Tenant/utils/queryTemplates.ts

```diff
@@ -42,6 +42,17 @@ CREATE TABLE \`${path}/ydb_column_table\` (
 PARTITION BY HASH(id)
 WITH (STORE = COLUMN)`;
 };
+export const createAsyncReplicationTemplate = () => {
+    return `CREATE ASYNC REPLICATION my_replication
+FOR <remote_table_name> AS <local_table_name> [, <remote_table_name> AS <local_table_name> ...]
+WITH (
+    ENDPOINT=<endpoint>, -- endpoint, e.g. grpcs://mydb.ydb.tech:2135/
+    DATABASE=<path_to_database>, --/cluster/dynamo
+    USER=<user>,
+    PASSWORD=<pwd>,
+    ...
+);`;
+};
```

Comment:
I don't really like `<>` in this template. In other template we use some preset values, so you can alter them and don't need to think of the form. For example - should I use some marks and which marks should I use - ', " or `. For me, it's better to have the form like:
```
WITH (
    ENDPOINT="grpcs://mydb.ydb.tech:2135/",
    DATABASE="your_database",
    USER="user",
    PASSWORD="your_password"
)
```

Reply:
agree with you!